### ISSUE TITLE
Error during "Estimate model parameters (Channels)"

### ISSUE THREAD

**Issue by AmZNeu** [LIMO-EEG-Toolbox/limo_tools#54]
Hi, 

I'm performing the first-level analysis of repeated measures. During the "Estimate model parameters (Channels)", this error appears: 

*Unable to read file "xxxx/xxxx/xxxxx/NM001.dataerp". No such file or directory.
Error occurred in function std_limo() at line 254"

The directory exists but the file created during the STUDY design, instead of being named  "NM001.dataerp" is named "NM001_ses-01.dataerp"

Is there a solution to fix the issue? 

Thanks in advance, 
A

**Comment by CPernet** [comment]
ok it's unusual - did you fill the session ID in this case? removing it should work - can you let me know here, thx
note session is not compatible with LIMO right now, but we are working on it

**Comment by AmZNeu** [comment]
Hi, 

thanks for your response. My design is a repeated measures ANOVA with 2 groups x 3 sessions and I had filled the session ID and group ID. Now, I changed "session" by "condition" and worked (i.e., it created the file named "NM001.dataerp", instead of "NM001_ses-01.dataerp"). However, a new error appears although the measures were computed using the STUDY functions of EEGLAB. 

"Error using std_limo (line 286)
std_limo subject NM001: Measures must be computed first"
